common_scripts/enigma_docker_common/provider.py

```diff
@@ -40,6 +42,14 @@ def __init__(self, config: dict):
             self._enigma_contract_abi_filename = config.get('ENIGMA_CONTRACT_ABI_FILENAME', 'Enigma.json')
             self._enigma_contract_abi_filename_zip = config.get('ENIGMA_CONTRACT_ABI_FILENAME_ZIPPED', 'Enigma.zip')
 
+        self._voting_abi_directory = config.get('VOTING_CONTRACT_ABI_DIRECTORY', 'voting')
+        self._voting_abi_filename = config.get('VOTING_CONTRACT_ABI_FILENAME', 'VotingETH.json')
```

Comment:
Any reason to have this here? You're cloning the built contracts into the client (so these are unused), and I don't think any other component in the system will ever need to download these contract ABIs

Reply:
I copy the ABIs before they are deployed, so they don't have the contract addresses. For consistency reasons, I'm treating the `Sample` and `VotingETH` contracts like `Enigma` and `EnigmaToken` in those instances where I will need them (in the integration tests in the COMPOSE environment. Note that it's precisely in the client Python code that I am only requesting these addresses if I'm in the COMPOSE environment.

Reply:
Getting the addresses from here is great, but my point was that you're creating placeholder APIs to get ABI files (VotingEth.json/Sample.Json) that are not only unused, but will actually fail, since they're pointing to an Azure file share and they aren't uploaded there. Unless there is a use-case I'm missing, I would avoid adding this

Reply:
removed 1cb172a817f7a75b378d4b04378a66f3ca7c4ecc